precode.sh

```diff
@@ -7,37 +7,70 @@
 #   dir3
 #       dir4
 
+mkdir task && mkdir task/dir{1..3} && mkdir task/dir3/dir4
```

Comment:
Команда `mkdir` можно принимать неограниченное количество аргументов, а флаг `-p` создает промежуточные папки, если их нет
```suggestion
mkdir -p task/dir{1..3} task/dir3/dir4
```

Reply:
Или 
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```